modules/c++/sys/source/OSUnix.cpp

```diff
@@ -210,6 +210,14 @@ std::string sys::OSUnix::getEnv(const std::string& s) const
     return std::string(p); 
 }
 
+bool sys::OSUnix::isEnvSet(const std::string& s) const
+{
+    const char* p = getenv(s.c_str());
+    if (p == NULL)
```

Comment:
Try to avoid single character variable names (though I'm guessing getEnv() did this).  Also can simplify the return stuff to a single line.  How about...

```
const char* const envVal  = getenv(s.c_str());
return (envVal != NULL);
```